Lib/tkinter/tix.py

```diff
@@ -24,10 +24,6 @@
 
 import os
 import tkinter
-from tkinter import *
```

Comment:
I would not touch `tix.py`. It is deprecated and may be removed in 3.9.

Reply:
Great news, it seemed like a mess :)